assets/css/v2/dup/free_text.scss

```diff
@@ -270,6 +270,7 @@
     border-radius: 72px;
     font-size: 96px;
     line-height: 117px;
```

Comment:
I am pro- `line-height: 128px`, as you mentioned over slack. I think it looks right. Do you still think it's off?

![Screenshot 2023-04-20 at 8 31 23 AM](https://user-images.githubusercontent.com/69368883/233366567-3bea67b6-f74d-4bf2-88fd-922e1c96bd50.png)

Reply:
Looks good to me!